### PR TITLE
Fix labels in post with clearer ones

### PR DESCRIPTION
**Pull request by dugajean**

### CODE REVIEW

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-global-transformations.php [hunk 1]
```diff
@@ -402,11 +402,15 @@ private function init_taxonomy_manager( $post ) {
...
-		$type  = get_post_meta( $post->ID, self::META_APPLY_KEY . '_terms', true );
-		$out[] = '<label class="cld-tax-order-list-type"><input ' . checked( 'overwrite', $type, false ) . ' type="checkbox" value="overwrite" name="cld_apply_type" />' . __( 'Overwrite', 'cloudinary' ) . '</label>';
+		if ( ! empty( $terms ) ) {
+			$type  = get_post_meta($post->ID, self::META_APPLY_KEY . '_terms', true);
+			$out[] = '<label class="cld-tax-order-list-type"><input ' . checked( 'overwrite', $type, false ) . ' type="checkbox" value="overwrite" name="cld_apply_type" />' . __('Overwrite taxonomy', 'cloudinary') . '</label>';
+		}
+		
 		$out[] = '</div>';
-
+
```
Comment (pereirinha) [inline cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-global-transformations.php:413]: @dugajean 

Can we remove these white spaces/tabs?

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-global-transformations.php [hunk 2]
```diff
@@ -402,11 +402,15 @@ private function init_taxonomy_manager( $post ) {
 			}
 		}
 		$out[] = '</ul>';
+		
 		// Get apply Type.
-		$type  = get_post_meta( $post->ID, self::META_APPLY_KEY . '_terms', true );
-		$out[] = '<label class="cld-tax-order-list-type"><input ' . checked( 'overwrite', $type, false ) . ' type="checkbox" value="overwrite" name="cld_apply_type" />' . __( 'Overwrite', 'cloudinary' ) . '</label>';
+		if ( ! empty( $terms ) ) {
+			$type  = get_post_meta($post->ID, self::META_APPLY_KEY . '_terms', true);
```
Comment (pereirinha) [inline cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-global-transformations.php:408]: @dugajean 

It seems that we are removing the spaces inside the parenthesis.

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-global-transformations.php [hunk 3]
```diff
@@ -402,11 +402,15 @@ private function init_taxonomy_manager( $post ) {
...
+			$out[] = '<label class="cld-tax-order-list-type"><input ' . checked( 'overwrite', $type, false ) . ' type="checkbox" value="overwrite" name="cld_apply_type" />' . __('Overwrite taxonomy', 'cloudinary') . '</label>';
```
Comment (pereirinha) [inline cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-global-transformations.php:409]: @dugajean 

Same here, in the translatable string.

**Review by pereirinha — COMMENTED** [review]
@dugajean 

Can you please check the minor comments?

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-global-transformations.php (hunk 1 above)
Comment (dugajean) [reply cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-global-transformations.php:413]:
```suggestion

```

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-global-transformations.php (hunk 2 above)
Comment (dugajean) [reply cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-global-transformations.php:408]:
```suggestion
			$type  = get_post_meta( $post->ID, self::META_APPLY_KEY . '_terms', true );
```

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-global-transformations.php (hunk 3 above)
Comment (dugajean) [reply cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-global-transformations.php:409]:
```suggestion
			$out[] = '<label class="cld-tax-order-list-type"><input ' . checked( 'overwrite', $type, false ) . ' type="checkbox" value="overwrite" name="cld_apply_type" />' . __( 'Overwrite taxonomy', 'cloudinary' ) . '</label>';
```

**Comment by pereirinha** [comment]
@dugajean 

Looks good to me and it's working fine.
Can you please fix the conflicts?

**Review by pereirinha — APPROVED** [review]
@dugajean 

The PR is approved, but we need to resolve the merge conflicts before.
Don't merge it once you resolve. I'll take care of that part.

**Pull request merged**